test/lib/analyzer/os-release-detector.test.ts

```diff
@@ -117,8 +128,11 @@ test('os release detection', async t => {
     sinon.match.any, 'cat', sinon.match.any,
   ])
     .callsFake(async (docker, [run, rm, entry, empty, network, none, image, cat, file]) => {
+      const example = examples[image];
+      if (example.dir === '') {
```

Comment:
Adding here an extra condition to make sure that all files in case of the scratch image resolve to "not found"